hwangsae/org.hwangsaeul.hwangsae.gschema.xml

```diff
@@ -11,6 +11,14 @@
       <summary>SRT binding port to be a source</summary>
       <description>SRT listening port to acquire stream</description>
     </key>
+    <key name="external-ip" type="s">
+      <default>"0.0.0.0"</default>
```

Comment:
This default will make `hwangsae_relay_get_sink_uri()` return `srt://0.0.0.0:8888`, which isn't a useful URI. I suggest we have an empty string as the default, so that the relay picks some local IP on its own.

I'm not sure if GSettings returns `NULL` or `""` for an empty string value. It might be a good idea to check for both cases in `hwangsae_relay_set_property()`:

```
case PROP_EXTERNAL_IP: {
  const gchar *ip = g_value_get_string (value);
  g_clear_pointer (&self->sink_uri, g_free);
  g_clear_pointer (&self->source_uri, g_free);
  g_clear_pointer (&self->external_ip, g_free);
  if (ip && ip[0] != '\0') {
    self->external_ip = g_strdup(ip);
  }
  break;
}
```